attachments/views.py

```diff
@@ -60,8 +69,26 @@ def attach(request, session_id):
                     session.data = {key: value}
             session.save()
             return JsonResponse({'ok': True, 'file_name': f.name, 'file_size': f.size}, content_type=content_type)
+        except FileSizeException as ex:
+            return JsonResponse({'ok': False, 'error': unicode(ex)}, content_type=content_type)
+        except FileTypeException as ex:
+            return JsonResponse({'ok': False, 'error': unicode(ex)}, content_type=content_type)
         except VirusFoundException as ex:
-            logger.exception(str(ex))
+            now = datetime.datetime.now()
+            now = now.strftime('%m-%d-%Y  %H:%M:%S')
+            user = getattr(request, 'user', None)
+            log_message = loader.render_to_string('virus_email.txt', 
```

Comment:
Shouldn't this be `attachments/virus_email.txt`? Same for virus_subject.txt below? Also, it doesn't seem that the `now` variable is used anywhere, can it be removed?